org.feichtmeier.Musicpod.json

```
@@ -0,0 +1,228 @@
+                                        "stable-only": true,
+                                        "url-template": "https://www.freedesktop.org/software/uchardet/releases/uchardet-$version.tar.xz"
+                                   }
+                              }
+                         ]
+                    },
+                    {
+                         "name": "squashfs-tools",
+                         "cleanup": ["*"],
+                         "buildsystem": "simple",
+                         "build-commands": [
+                              "XZ_SUPPORT=1 LZO_SUPPORT=0 make -C squashfs-tools -j ${FLATPAK_BUILDER_N_JOBS} install INSTALL_DIR=${FLATPAK_DEST}/bin INSTALL_MANPAGES_DIR=${FLATPAK_DEST}/man/man1"
+                         ],
+                         "sources": [
+                              {
+                                   "type": "git",
+                                   "url": "https://github.com/plougher/squashfs-tools.git",
+                                   "tag": "4.6.1",
+                                   "commit": "d8cb82d9840330f9344ec37b992595b5d7b44184",
+                                   "x-checker-data": {
+                                        "type": "git",
+                                        "tag-pattern": "^([\\d.]+)$"
+                                   }
+                              }
+                         ]
+                    },
+                    {
+                         "name": "musicpod",
+                         "buildsystem": "simple",
+                         "build-commands": [
+                              "unsquashfs -quiet -no-progress musicpod.snap",
+                              "cp -r squashfs-root/bin /app",
+                              "install -Dm644 squashfs-root/snap/gui/musicpod.desktop /app/share/applications/org.feichtmeier.Musicpod.desktop",
+                              "desktop-file-edit --set-icon='org.feichtmeier.Musicpod' /app/share/applications/org.feichtmeier.Musicpod.desktop",
+                              "install -Dm644 squashfs-root/snap/gui/musicpod.png /app/share/icons/hicolor/256x256/apps/org.feichtmeier.Musicpod.png",
+                              "install -Dm644 org.feichtmeier.Musicpod.appdata.xml /app/share/metainfo/org.feichtmeier.Musicpod.appdata.xml"
+                         ],
+                         "cleanup": [
+                              "/include",
+                              "/lib/pkgconfig",
+                              "/share/man"
+                         ],
+                         "sources": [
+                              {
+                                   "type": "file",
+                                   "sha256": "aec5891c54a81ab64799b1e9efa05899d2ae16503313168152dae80f25523ce2",
+                                   "url": "https://raw.githubusercontent.com/ubuntu-flutter-community/musicpod/d6f06d660fd5bf143425b79f5f8a93d328b34de3/flatpak/org.feichtmeier.Musicpod.appdata.xml"
```

Do I have to change the sha256 too?

```suggestion
                                   "url": "https://github.com/ubuntu-flutter-community/musicpod/blob/125f383f759e5d6e35f77ced2f8f65c853072305/flatpak/org.feichtmeier.Musicpod.appdata.xmll"
```

---

Yes if the file changes the hash changes too

---

Keep the raw.githubusercontent link.

---

Made the fix.

@Feichtmeier I think these are ready, but someone might want to check my sha256.